src/main/java/org/sonar/scm/git/blame/diff/ContentSource.java

```diff
@@ -0,0 +1,340 @@
+/*
+ * Copyright (C) 2010, 2021 Google Inc. and others
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Distribution License v. 1.0 which is available at
+ * https://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+package org.sonar.scm.git.blame.diff;
```

Comment:
Is there a way to make it clear that those files are just ported and not supposed to be modified?

Reply:
good idea. Maybe a comment after the header?

Reply:
Yes I guess that could work

Reply:
done